api-specificatie/Bevraging-Ingeschreven-Persoon/openapi.yaml

```diff
@@ -1849,8 +1887,8 @@ components:
       properties:
         self:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/components.yaml#/HalLink"
-        ingeschrevenpersonen:
-          title: "isPartner"
+        isIngeschrevenPersoon:
+          title: "isingeschrevenPersoon"
```

Comment:
Ik zou het ingeschrevenPersoon noemen. Bij isIngeschrevenPersoon verwacht ik eerder een Boolean (ja/nee of true/false) als resultaat dan een persoon